src/Sylius/Component/Core/Model/Order.php

```diff
@@ -529,9 +529,8 @@ public function getShippingTotal(): int
     {
         $shippingTotal = $this->getAdjustmentsTotal(AdjustmentInterface::SHIPPING_ADJUSTMENT);
         $shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::ORDER_SHIPPING_PROMOTION_ADJUSTMENT);
-        $shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::TAX_ADJUSTMENT);
 
-        return $shippingTotal;
+        return $shippingTotal + $this->getAdjustmentsTotal(AdjustmentInterface::TAX_ADJUSTMENT);
```

Comment:
I'd say it's counterintuitive here.

Reply:
Why exactly?

Reply:
I see. The main goal is readability, not as short syntax as possible (oneliner).

So having 3 visually similar edits make it more readable than 2 and 1 short one, even with the same operation? I agree with that and I didn't take it into account before.

I'll try to put this into AST.
And revert this